Arrowgene.Ddon.Shared/Crypto/Camellia.cs

```diff
@@ -404,14 +413,17 @@ private void or4octets(Span<byte> src1, Span<byte> src2, Span<byte> dst)
         /* x[] <<<= 1 */
         private void rot1(int nOctets, Span<byte> x)
         {
-            byte x0 = x[0];
-            nOctets--;
-            for (int i = 0; i < nOctets; i++)
+            unchecked
```

Comment:
Well, this is part of the Shared project. As far as I can tell, I can only set the flag at that level, so that is why I needed to add these checks in this particular subproject.